Source/WebKit/UIProcess/API/glib/WebKitCookieManager.cpp

```diff
@@ -517,3 +517,105 @@ void webkit_cookie_manager_delete_all_cookies(WebKitCookieManager* manager)
     webkit_website_data_manager_clear(manager->priv->dataManager, WEBKIT_WEBSITE_DATA_COOKIES, 0, nullptr, nullptr, nullptr);
 }
 #endif
+
+/**
+ * webkit_cookie_manager_set_cookie_jar:
+ * @cookie_manager: a #WebKitCookieManager
+ * @cookies: (element-type SoupCookie) (transfer full): A #GList of #SoupCookie to be added
```

Comment:
Note that the `WebCore::Cookie` constructor only reads fields from the `SoupCookie`, making copies of values. Therefore this should be `(transfer none)` because the function is _not_ taking ownership of (and not freeing, nor consuming) the passed list of cookies.